README.md

```diff
@@ -148,24 +150,27 @@ See [CONTRIBUTING.md](CONTRIBUTING.md).
 There are also community tools that use OSV. Note that these are community built
 tools and unsupported by the core OSV maintainers.
 
+-   [Betterscan.io: Code Scanning/SAST/Static Analysis/Linting using many
+    tools/Scanners with One Report (Code,
+    IaC)](https://github.com/marcinguy/betterscan-ce)
+-   [bomber](https://github.com/devops-kung-fu/bomber)
+-   [Cortex XSOAR](https://github.com/demisto/content)
+-   [Dependency-Track](https://github.com/DependencyTrack/dependency-track)
+-   [dep-scan](https://github.com/AppThreat/dep-scan)
+-   [EZE-CLI: The one stop shop for security testing in modern development](https://github.com/RiverSafeUK/eze-cli)
+-   [Golang support for the schema](https://pkg.go.dev/golang.org/x/vuln/osv)
 -   [G-Rath/osv-detector](https://github.com/G-Rath/osv-detector): A scanner
     that uses the OSV database.
--   [Rust client library](https://github.com/gcmurphy/osv)
--   [Golang support for the schema](https://pkg.go.dev/golang.org/x/vuln/osv)
--   [pip-audit](https://pypi.org/project/pip-audit/)
+-   [it-depends](https://github.com/trailofbits/it-depends)
 -   [.NET client library and support for the schema](https://github.com/JamieMagee/osv.net)
--   [EZE-CLI: The one stop shop for security testing in modern development](https://github.com/RiverSafeUK/eze-cli)
 -   [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort)
+-   [Packj](https://github.com/ossillate-inc/packj)
```

Comment:
I wonder if we should order this list by star instead, so the most popular tools are listed first.

Reply:
That would be an interesting exercise to implement a sorter for...